mobile-widgets/qml/DownloadFromDiveComputer.qml

```diff
@@ -99,12 +99,12 @@ Kirigami.Page {
 			Controls.ComboBox {
 				id: comboProduct
 				Layout.fillWidth: true
-				Layout.preferredHeight: fontMetrics.height * 2.5
+				Layout.preferredHeight: Kirigami.Units.gridUnit * 2.5
 				model: null
 				currentIndex: -1
 				delegate: Controls.ItemDelegate {
 					width: comboProduct.width
-					height: fontMetrics.height * 2.0
+					height: Kirigami.Units.gridUnit * 2.5
```

Comment:
I notice that this is the only one where you change the factor... cut and paste error or intentional?

Reply:
No I looked at other examples which uses 2.5

I have no problem using kirigami.units.* but all these different factor, is not something I like. I am thinking at making a number of global properties like "standardHeight" etc.